libs/core/cornflow_core/schemas/query.py

```diff
@@ -11,3 +11,14 @@ class BaseQueryFilters(Schema):
     offset = fields.Int(required=False, dump_default=0)
     creation_date_gte = fields.DateTime(required=False)
     creation_date_lte = fields.DateTime(required=False)
+    id = fields.Str(required=True)
+    name = fields.Str(required=True)
+    deletion_date_gte = fields.DateTime(required=False)
+    deletion_date_lte = fields.DateTime(required=False)
+    update_date = fields.DateTime(required=False)
```

Comment:
The `update_date` should allow `gte` and `lte` as the `creation_date` and `deletion_date`